src/docs/build-and-earn/testnet-guides/run-seed-node.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Run a Seed Node
+---
+
+## ﻿Run a Seed Node
+
+Before running a node, make sure that you have followed the guide for [setting up your prerequisites and environment](/docs/build-and-earn/testnet-guides/prerequisites.md).
```

Comment:
Link [setting up your prerequisites and environment] is not working. when you click on the link it will take you to 404 page of github